superset-frontend/plugins/plugin-chart-table/src/Styles.tsx

```diff
@@ -32,10 +32,12 @@ export default styled.div`
     td {
       min-width: 4.3em;
     }
-
     thead > tr > th {
+      position: sticky;
+      top: -1px;
       padding-right: 0;
-      position: relative;
+      z-index: 100;
+      border-bottom: 2px solid #e0e0e0;
```

Comment:
```suggestion
      border-bottom: ${theme.gridUnit /2}px solid ${theme.colors.grayscale.light2};
```
you'll want to use the theme here for styling when possible